editor/constants.js

```diff
@@ -0,0 +1,10 @@
+import scssVariables from '!!sass-variables-loader!./assets/stylesheets/_variables.scss';
```

Comment:
Interesting loader. Why do you need the `!!` prefix?

Also, for consistency we should add the `Internal dependencies` DocBlock.

Reply:
The !! disables other configured loaders during the import process. Without this flag or import was not being successful.